src/web/routes/application/manual-address/manual-address.js

```diff
@@ -32,13 +32,18 @@ const contentSummary = (req) => ({
   ])
 })
 
+const isNavigable = (session) => {
+  return isNil(session.claim.selectedAddress)
```

Comment:
I'm not sure this is going to work. What if someone selects an address in the dropdown, then decides that it's not the correct address and clicks 'my address is not listed'?

Reply:
This is handled in the behaviourForGet